src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -454,7 +479,20 @@ public override bool CanVerifySignedPackages(SignedPackageVerifierSettings verif
             }
             else if (RuntimeEnvironmentHelper.IsLinux || RuntimeEnvironmentHelper.IsMacOSX)
             {
-                return false;
+                // Conditionally enable back package sign verification temporary disabled due to Mozilla drop Symantec as CA on Linux/MAC.
+                // Please note: Linux/MAC case sensitive for env var.
+                string signVerifyEnvVariable = _environmentVariableReader == null ?
+                    Environment.GetEnvironmentVariable("DOTNET_OPT_IN_SECURE_PACKAGE_VERIFICATION") :
```

Comment:
``` suggestion
EnvironmentVariableWrapper.Instance.GetEnvironmentVariable("DOTNET_OPT_IN_SECURE_PACKAGE_VERIFICATION") :
```
With this approach, any `SecurityException` raised by `Environment.GetEnvironmentVariable` will be handled by the instance.

Reply:
Ok. Addressed.